hrm-service/src/app.js

```diff
@@ -43,6 +43,11 @@ const canAccessResourceWithStaticKey = (path, method) => {
   return false;
 };
 
+const isWorker = tokenResult =>
+  Boolean(tokenResult.worker_sid) && tokenResult.worker_sid.startsWith('WK');
+const isGuest = tokenResult =>
+  Array.isArray(tokenResult.roles) && tokenResult.roles.includes('guest');
```

Comment:
There's no possibility that a user could have this role as well as other ones that give them legitimate access? i.e. there's no way a legit counsellor could somehow have the guest role causing them to be blocked out?